.env.development

```diff
@@ -84,6 +84,7 @@ NUXT_ENV_WALLETCONNECT_BRIDGE_URL="https://uniswap.bridge.walletconnect.org"
 
 NUXT_ENV_MASTERCHEF_CONTRACT=0xE2F0627DCA576CCdbce0CED3E60E0E305b7D4E33
 NUXT_ENV_EFX_TOKEN_CONTRACT=0xC51Ef828319b131B595b7ec4B28210eCf4d05aD0
+NUXT_ENV_BNB_TOKEN_CONTRACT=0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c
```

Comment:
This same value has to be added to `.env.production`

Reply:
Change has been completed